lib/resources/forms.js

```diff
@@ -69,6 +69,13 @@ module.exports = (service, endpoint) => {
     /* eslint-enable indent */
       // if we don't have managed encryption, or the form carries its own key,
       // we can use the form xml as-is. otherwise we must inject things.
```

Comment:
I think this comment should now move down to the next `.then`?

Reply:
ah yes, moved down!